alacritty/src/event.rs

```diff
@@ -272,7 +272,7 @@ pub struct Mouse {
     pub right_button_state: ElementState,
     pub last_click_timestamp: Instant,
     pub click_state: ClickState,
-    pub scroll_px: i32,
+    pub scroll_px: f32,
```

Comment:
Might as well go for `f64` here, since that's the highest precision we receive.

Reply:
Also was thinking about that. Wasnt sure though. (updated now)